### PR TITLE
chore: add ninja to dx roadmap

### PR DESCRIPTION
**Pull request by dfx-json**
Adds [ICP Ninja](icp.ninja) to the Developer Experience category in the ICP Roadmap

### CODE REVIEW

File: roadmap/entries/data-dx.json [hunk 1]
```diff
@@ -2,6 +2,16 @@
   "categoryName": "Developer Experience",
   "categoryDescription": "Developer experience is an essential ingredient for developer adoption. This includes low-friction canister development and operations, expressive smart contract languages, testing frameworks, and a rich set of libraries. Beyond canister development, developer experience also encompasses an inclusive and quality-preserving protocol development.",
   "elements": [
+    {
+      "title": "ICP Ninja",
+      "overview": "ICP Ninja is your go-to online platform for developing and deploying smart contracts on the Internet Computer (ICP)",
+      "status": "deployed",
+      "forum": "https://forum.dfinity.org/t/announcing-icp-ninja/36396",
+      "proposal": "",
+      "docs": "",
```
Comment (jessiemongeon1) [inline roadmap/entries/data-dx.json:11]: Do we want to add the link to the Ninja dev docs page?
https://internetcomputer.org/docs/current/developer-docs/developer-tools/ide/icp-ninja

File: roadmap/entries/data-dx.json
Comment (MarcoT0S) [inline roadmap/entries/data-dx.json:1]: It doesn't render on the website preview (https://7zohq-kyaaa-aaaam-abosa-cai.icp0.io/roadmap), you are using the structure and fields of a feature, you should create ICP.Ninja as a Milestone.

See https://docs.google.com/document/d/19-7sbrOB1uWJsrgI6AAaJGIqV31QeaKt0QWA3bIKKOI/edit?tab=t.0#heading=h.6nz2h53fvqoz

Also, when creating it as a milestone, it should contain at least 1 feature.

As milestone name, Coulomb and Flux are still available from the initial name list but any name related to Fusion Technology will work.

Comment (MarcoT0S) [inline roadmap/entries/data-dx.json:1]: Can you please move the new milestone after Beryllium in the file? 
It seems the ordering in this file effects how it's rendered on the roadmap (I just realized it looking at the preview of the change).

File: roadmap/entries/data-dx.json [hunk 2]
```diff
@@ -82,7 +82,60 @@
...
+      "is_milestone": true,
+      "description": "This milestone introduces new products that make building on ICP fast, fun, and educational.",
+      "milestone_id": "Coulomb",
+      "eta": "",
+      "status": "in_progress",
+      "eta_to_render": ""
+    },
+    {
+      "title": "ICP Ninja BETA",
+      "overview": "ICP Ninja is your go-to online platform for developing and deploying smart contracts on the Internet Computer (ICP)",
```
Comment (jessiemongeon1) [inline roadmap/entries/data-dx.json:98]:
```suggestion
      "overview": "ICP Ninja is your go-to online platform for developing and deploying smart contracts on the Internet Computer (ICP).",
```

File: roadmap/entries/data-dx.json [hunk 3]
```diff
@@ -82,7 +82,60 @@
...
+      "forum": "https://forum.dfinity.org/t/announcing-icp-ninja/36396",
+      "proposal": "",
+      "docs": "https://internetcomputer.org/docs/current/developer-docs/developer-tools/ide/icp-ninja",
+      "is_community": false,
+      "in_beta": true,
+      "milestone_id": "Coulomb"
+    },
+    {
+      "title": "ICP Ninja User Experience Enhancements",
+      "overview": "ICP Ninja will receive a number of new user experience enhancements, such as dark mode and the ability to modify the file tree",
```
Comment (jessiemongeon1) [inline roadmap/entries/data-dx.json:109]:
```suggestion
      "overview": "ICP Ninja will receive a number of new user experience enhancements, such as dark mode and the ability to modify the file tree.",
```

**Review by jessiemongeon1 — APPROVED** [review]

**Pull request merged**